wman-frontend/src/components/Worker-page/one-day-event/one-day-events.component.jsx

```diff
@@ -0,0 +1,83 @@
+import react, { useState } from "react";
+import OneLabel from "../one-label.component.jsx/one-label.component";
+import "./one-day-events-styles.css"
+import Moment from "react-moment";
+
+const OneDayEvents = (event) => {
+
+    const [show, setShow] = useState(false);
+    const toggleShow = () => {
+        setShow(!show);
+    }
+    console.log();
+
+    return (
+        <div className='one-event' onClick={toggleShow}>
+            <div className='one-event-tables'>
+                <table>
+                    <tr>
+                        <th colSpan='2' ><b>Description:</b></th>
+                    </tr>
+                    <tr>
+                        <th colSpan='2' >{event.event.jobDescription}</th>
+                    </tr>
+                    <tr>
+                        <th>Start:</th>
+                        <th>
+                            <Moment format="YYYY.MM.DD. HH:MM">{event.event.estimatedStartDate}</Moment>
+                        </th>
+                    </tr>
+                    <tr>
+                        <th>Finish:</th>
+                        <th> <Moment format="YYYY.MM.DD. HH:MM">{event.event.estimatedFinishDate}</Moment></th>
```

Comment:
The minutes component should be lower case in order to display the minutes instead of the month. This applies to the code at line 27 too.
```suggestion
                        <th> <Moment format="YYYY.MM.DD. HH:mm">{event.event.estimatedFinishDate}</Moment></th>
```